app/main.py

```diff
@@ -1,3 +1,17 @@
 def cache(func):
-    # Write your code here
-    pass
+    cached_values = []
+
+    def inner(*args):
+
+        lst = [*args]
+
+        if lst in cached_values:
+            print("Getting from cache")
+        else:
+            print("Calculating new result")
+
+        cached_values.append([*args])
+
+        return func(*args)
```

Comment:
You always calculate a new value and don't use the value calculated before. There is no point in that decorator, rewrite.